src/battle_script_commands.c

```diff
@@ -1748,7 +1749,7 @@ u32 GetTotalAccuracy(u32 battlerAtk, u32 battlerDef, u32 move, u32 atkAbility, u
     return calc;
 }
 
-static void Cmd_accuracycheck(void)
+static void AccuracyCheck(bool32 recalcDragonDarts)
 {
     CMD_ARGS(const u8 *failInstr, u16 move);
```

Comment:
I think it would be better to keep `CMD_ARGS` inside `Cmd_` functions, and just pass these two as arguments to `AccuracyCheck`.